### PR TITLE
20 12 2017

### PR DESCRIPTION
**Pull request by githubwebcontentor**

### CODE REVIEW

File: WebApp/App_Code/BusinessLogic/Configuration.cs [hunk 1]
```diff
@@ -704,7 +704,7 @@ private bool ValidateApplications()
 
       if (application.IsValidationErrorNull() && !application.IsDefaultToolNull())
       {
-        string[] tools = new string[] { "pan", "identify", "select", "drawlength", "drawarea", "drawpoint", "drawline", "drawpolygon", "drawcircle", "drawtext", "drawcoordinates", "deletemarkup", "colorpicker", "paintbucket" };
+				string[] tools=new string[] { "identify", "select", "drawlength", "drawarea", "drawpoint", "drawline", "drawpolygon", "drawcircle", "drawtext", "drawcoordinates", "deletemarkup", "colorpicker", "paintbucket" };
```
Comment (githubwebcontentor) [inline WebApp/App_Code/BusinessLogic/Configuration.cs]: indent this file

File: WebApp/App_Code/Services/MarkupPanelHandler.cs [hunk 2]
```diff
@@ -77,7 +77,7 @@ private void CreateMarkupGroup()
 
     int id = Sequences.NextMarkupGroupId;
     string title = "[untitled]";
-    string details = "[unDetailed]";
+    string details = "[enter Details]";
```
Comment (githubwebcontentor) [inline WebApp/App_Code/Services/MarkupPanelHandler.cs]: No Details

File: WebApp/BaseMap.ascx.cs [hunk 3]
```diff
@@ -56,6 +56,8 @@ private void AddTiles(Configuration.MapTabRow mapTabRow, AppState appState)
       name.Attributes["class"] = "LegendName";
       name.InnerText = tileGroupRow.DisplayName;
     }
+
```
Comment (githubwebcontentor) [inline WebApp/BaseMap.ascx.cs]: Remove extra line

File: WebApp/BaseMap.ascx.cs [hunk 4]
```diff
@@ -64,7 +66,33 @@ public void Initialize(Configuration config, AppState appState, Configuration.Ap
     {
       Configuration.MapTabRow mapTabRow = appMapTabRow.MapTabRow;
       AddTiles(mapTabRow, appState);
+			HtmlGenericControl parentLegend=new HtmlGenericControl("div");
```
Comment (githubwebcontentor) [inline WebApp/BaseMap.ascx.cs]: indent this file

File: WebApp/Development/CompiledIcons.ashx [hunk 5]
```diff
@@ -29,7 +29,7 @@ public class CompiledIcons : IHttpHandler
   
   public void ProcessRequest(HttpContext context)
   {
-    string imagePath = context.Server.MapPath("~/Development/Images/UI/Icons");
+    string imagePath = context.Server.MapPath("~/Development/Images/Icons");
```
Comment (githubwebcontentor) [inline WebApp/Development/CompiledIcons.ashx]: why this changed from UI.

File: WebApp/Header.ascx [hunk 6]
```diff
@@ -16,5 +16,5 @@
 
 <%@ Control Language="C#" ClassName="Header" %>
   <div class="logo">
-    <img src="Images/UI/AppGeo.png" />
+   <%-- <img src="Images/UI/AppGeo.png" />--%>
```
Comment (githubwebcontentor) [inline WebApp/Header.ascx]: delete this line

File: WebApp/LegendPanel.ascx [hunk 7]
```diff
@@ -17,13 +17,13 @@ limitations under the License.
 <%@ Control Language="C#" AutoEventWireup="true" CodeFile="LegendPanel.ascx.cs" Inherits="LegendPanel" %>
 <%@ Register TagPrefix="gpv" Assembly="App_Code" Namespace="GPV" %>
 
-<div class="topHead">
+<div class="topHead FunctionHeader">
```
Comment (githubwebcontentor) [inline WebApp/LegendPanel.ascx]: What is ```FunctionHeader``` and why it moved to the above element.

File: WebApp/LegendPanel.ascx.cs [hunk 8]
```diff
@@ -300,48 +300,6 @@ private void AddLayers(Configuration.MapTabRow mapTabRow, AppState appState)
     }
   }
 
-  // removing this fron viwer page
```
Comment (githubwebcontentor) [inline WebApp/LegendPanel.ascx.cs:303]: What is this code for?

File: WebApp/LocationPanel.ascx [hunk 9]
```diff
@@ -16,13 +16,13 @@ limitations under the License.
 
 <%@ Control Language="C#" AutoEventWireup="true" CodeFile="LocationPanel.ascx.cs" Inherits="LocationPanel" %>
 <%@ Register TagPrefix="gpv" Assembly="App_Code" Namespace="GPV" %>
-<div class="topHead">
+<div class="topHead FunctionHeader">
```
Comment (githubwebcontentor) [inline WebApp/LocationPanel.ascx]: ``` FunctionHeader``` comment

File: WebApp/MarkupPanel.ascx [hunk 10]
```diff
@@ -14,13 +14,13 @@ limitations under the License.
 <%@ Control Language="C#" AutoEventWireup="true" CodeFile="MarkupPanel.ascx.cs" Inherits="MarkupPanel" %>
 <%@ Register TagPrefix="gpv" Assembly="App_Code" Namespace="GPV" %>
 
-<div class="topHead">
+<div class="topHead FunctionHeader">
```
Comment (githubwebcontentor) [inline WebApp/MarkupPanel.ascx:17]: ``` FunctionHeader ``` comment

File: WebApp/MarkupPanel.ascx [hunk 11]
```diff
@@ -43,16 +43,22 @@ limitations under the License.
...
+            <li id="optDrawCircle" runat="server" clientIDMode="static" class=" MapTool "><a href="#"><span class="imgflag draw-circle"></span>Draw Circle</a></li>
+            <li id="optDrawText" runat="server" clientIDMode="static" class=" MapTool "><a href="#"><span class="imgflag draw-text"></span>Draw Text</a></li>
+            <li id="optDrawCoordinates" runat="server" clientIDMode="static" class=" MapTool "><a href="#"><span class="imgflag draw-coordinates"></span>Draw Coordinates</a></li>
+             <li id="optDeleteMarkup" runat="server" clientIDMode="static" class=" MapTool "><a href="#"><span class="imgflag delete-markup"></span>Delete Markup</a></li>
+            <li id="optColorPicker" runat="server" clientIDMode="static" class=" MapTool  "><a href="#"><span class="imgflag color-picker"></span>Pick Color</a></li>
+            <li id="optPaintBucket" runat="server" clientIDMode="static" class="MapTool "><a href="#"><span class="imgflag paint-bucket"></span>Fill With Color</a></li>
+       <li id="optMeasureTitle" runat="server" clientIDMode="static" class=" toolTitle">Measure Tools - </li>
```
Comment (githubwebcontentor) [inline WebApp/MarkupPanel.ascx]: indent

File: WebApp/MarkupPanel.ascx [hunk 12]
```diff
@@ -103,20 +109,20 @@ limitations under the License.
...
+          <input type="text" id="tboMarkupTitle" class="frmInput btnControlLock" disabled="disabled" placeholder="Title" title="Title of the current markup group" /><br />
         </div>
       </div>
       <div class="frm_row">
         <div class="frmLabel">
           <span class="">Details</span>
         </div>
         <div class="frmField">
-          <textarea id="tblMarkupDetails" class="frmTextarea" placeholder="Details" title="Detail of the current markup group"></textarea>
+          <textarea id="tboMarkupDetails" class="frmTextarea" placeholder="Details" title="Detail of the current markup group"></textarea>
```
Comment (githubwebcontentor) [inline WebApp/MarkupPanel.ascx:125]: is this id used any where? if not remove this from the element.

File: WebApp/Scripts/BaseLayer.js [hunk 13]
```diff
@@ -13,16 +13,20 @@
 //  limitations under the License.
 
 var GPV = (function (gpv) {
-
   $(function () {
+    var previousChecked;
```
Comment (githubwebcontentor) [inline WebApp/Scripts/BaseLayer.js]: ```var previousChecked = null; ```

File: WebApp/Scripts/BaseLayer.js [hunk 14]
```diff
@@ -52,6 +60,10 @@ var GPV = (function (gpv) {
       getVisibleLayers: getVisibleLayers,
       getVisibleTiles: getVisibleTiles
     };
+   // alert(document.getElementsByName('ucBaseMapPanel$'));
```
Comment (githubwebcontentor) [inline WebApp/Scripts/BaseLayer.js]: Remove if not required

File: WebApp/Scripts/MarkupPanel.js [hunk 15]
```diff
@@ -181,42 +181,47 @@ var GPV = (function (gpv) {
...
+    $("#optDrawLength").on("click", function () {
 
       var c = getMarkupColor();
       gpv.selectTool($(this), map, { cursor: 'crosshair', drawing: { mode: "polyline", style: { color: c, fill: false } }, doubleClickZoom: false });
     });
 
-    $("#ucMarkupPanel_optDrawLine").on("click", function () {
+    var $optDrawLine = document.getElementById("optDrawLine");
+ 
+
```
Comment (githubwebcontentor) [inline WebApp/Scripts/MarkupPanel.js]: remove extra line

File: WebApp/Scripts/MarkupPanel.js [hunk 16]
```diff
@@ -227,13 +232,14 @@ var GPV = (function (gpv) {
       gpv.selectTool($(this), map, { cursor: 'crosshair', drawing: { mode: "polyline", style: { color: c, fill: false } }, doubleClickZoom: false });
     }
 
-    $("#ucMarkupPanel_optDrawPolygon").on("click", function () {
+ $("#optDrawPolygon").on("click", function () {
```
Comment (githubwebcontentor) [inline WebApp/Scripts/MarkupPanel.js]: indent this

File: WebApp/Scripts/MarkupPanel.js [hunk 17]
```diff
@@ -227,13 +232,14 @@ var GPV = (function (gpv) {
...
       var c = getMarkupColor();
       gpv.selectTool($(this), map, { cursor: 'crosshair', drawing: { mode: "polygon", style: { color: c, fill: true, fillColor: c } }, doubleClickZoom: false });
     });
 
     // =====  private functions  =====
 
+
```
Comment (githubwebcontentor) [inline WebApp/Scripts/MarkupPanel.js:239]: Remove extra line

File: WebApp/Scripts/MarkupPanel.js [hunk 18]
```diff
@@ -759,14 +765,14 @@ var GPV = (function (gpv) {
...
     // ==== for updating markup data ====
 
-    function updateMarkupGroupTitle() {
+    function UpdateMarkupGroupTitleAndDetails() {
       if (appState.MarkupGroups.length == 1) {
         var id = appState.MarkupGroups[0];
-        var title = $tblMarkupTitle.val();
-        var details = $('#tblMarkupDetails').val();
+        var title = $tboMarkupTitle.val();
+        var details = $('#tboMarkupDetails').val();
```
Comment (githubwebcontentor) [inline WebApp/Scripts/MarkupPanel.js]: use consistent naming convention 
```js
 var title = **$tboMarkupTitle**.val();
 var details = **$('#tboMarkupDetails')**.val();

```

File: WebApp/SelectionPanel.ascx [hunk 19]
```diff
@@ -18,13 +18,13 @@ limitations under the License.
 <%@ Register TagPrefix="gpv" Assembly="App_Code" Namespace="GPV" %>
 
 
-<div class="topHead">
+<div class="topHead FunctionHeader">
```
Comment (githubwebcontentor) [inline WebApp/SelectionPanel.ascx]: ```FunctionHeader``` comment

File: WebApp/SharePanel.ascx [hunk 20]
```diff
@@ -17,13 +17,13 @@ limitations under the License.
 <%@ Control Language="C#" AutoEventWireup="true" CodeFile="SharePanel.ascx.cs" Inherits="SharePanel" %>
 <%@ Register TagPrefix="gpv" Assembly="App_Code" Namespace="GPV" %>
 
-<div class="topHead">
+<div class="topHead FunctionHeader">
```
Comment (githubwebcontentor) [inline WebApp/SharePanel.ascx]: ```FunctionHeader``` comment why not removed from here.

File: WebApp/SharePanel.ascx [hunk 21]
```diff
@@ -45,7 +45,7 @@ limitations under the License.
   </div>
   <div id="pnlPrint" class="share">
     <div class="share_box">
-      <span class="seg_title">Create a Printable PDF Map</span>
+      <span class="FunctionLabel">Create a Printable PDF Map</span>
```
Comment (githubwebcontentor) [inline WebApp/SharePanel.ascx:47]: ```seg_title``` to ```FunctionLabel``` ??

throughout the file.

File: WebApp/Styles/Customize.css [hunk 22]
```diff
@@ -73,17 +75,17 @@
 
 /* default background color of the header and footer */
 
-#pnlHeader, #pnlFooter {
+#pnlHeader{
+    background-color: #3c8dbc;
```
Comment (githubwebcontentor) [inline WebApp/Styles/Customize.css]: Same property overwritten check line 43;

File: WebApp/Styles/Customize.css [hunk 23]
```diff
@@ -120,10 +122,10 @@ a.VersionText {
 
     /* background and text colors of tabs as you mouse over them */
 
-    .Tab.Normal:hover, a.Tab.Normal:hover {
+.Tab.Normal:hover, a.Tab.Normal:hover {
```
Comment (githubwebcontentor) [inline WebApp/Styles/Customize.css]: indent

File: WebApp/Styles/Customize.css [hunk 24]
```diff
@@ -159,77 +161,71 @@ a.VersionText {
...
-/* background and border colors of the map theme, level and tool dropdown buttons */
+/* color for function menu left boarder */
 
-.MapMenu > .dropdown-toggle {
-    text-align: left;
-        padding-left: 8px;
-    /*  background-color: White;
-  border-color: #cccccc;*/
+.Menu 
+{
```
Comment (githubwebcontentor) [inline WebApp/Styles/Customize.css]: should go above line

File: WebApp/Styles/Customize.css [hunk 25]
```diff
@@ -242,21 +238,26 @@ a.VersionText {
 /* data panel header background color and text color  */
 
 .DataHeader {
-  /*  background-color: #FFB74D;
-    color: #FFFFFF;*/
+    background-color: #172936;
+    color: red;
```
Comment (githubwebcontentor) [inline WebApp/Styles/Customize.css]: use Hexadecimal format

File: WebApp/Styles/Customize.css [hunk 26]
```diff
@@ -274,17 +275,16 @@ li.MapTool:hover, #selectMapTheme li:hover, #selectMapLevel li:hover {
 
 /* color of the overview map locator box that shows current extent */
 
-#locatorBox {
-    border: solid 2px #FF0000;
+#locatorBox 
+{
```
Comment (githubwebcontentor) [inline WebApp/Styles/Customize.css]: go above

File: WebApp/Styles/DataList.css [hunk 27]
```diff
@@ -28,11 +28,25 @@
 {
 }
 
-.DataList .RowSetHeader, .DataList .RowSetSubheader
+.DataList .RowSetHeader
 {
```
Comment (githubwebcontentor) [inline WebApp/Styles/DataList.css]: go above

File: WebApp/Styles/DataList.css [hunk 28]
```diff
@@ -91,7 +105,7 @@ Date 12-09-2017
 */
 /* table styles */
 .dataTable{
-    text-align: center;
+    /*text-align: center;*/
```
Comment (githubwebcontentor) [inline WebApp/Styles/DataList.css]: remove comments

File: WebApp/Styles/DataList.css [hunk 29]
```diff
@@ -112,7 +126,7 @@ Date 12-09-2017
 }
 
  thead th{
-    text-align: center;
+    /*text-align: center;*/
```
Comment (githubwebcontentor) [inline WebApp/Styles/DataList.css]: remove comments

File: WebApp/Styles/LegendPanel.css [hunk 30]
```diff
@@ -12,133 +12,110 @@
...
   height: 80%;
-  color:white;
+  color: white;
 }
 
-#pnlBaseMaps
-{
- /* position: absolute;
+#pnlBaseMaps {
+  /* position: absolute;
```
Comment (githubwebcontentor) [inline WebApp/Styles/LegendPanel.css]: remove comments

File: WebApp/Styles/LegendPanel.css [hunk 31]
```diff
@@ -12,133 +12,110 @@
...
-.LegendTop
-{
+.LegendTop {
   left: 8px;
   top: 10px;
 }
 
-.LegendEntry, .LegendHeader, .LegendContent, .LegendClass
-{
+.LegendEntry, .LegendHeader, .LegendContent, .LegendClass {
```
Comment (githubwebcontentor) [inline WebApp/Styles/LegendPanel.css:56]: check indentation in complete file

File: WebApp/Styles/LocationPanel.css [hunk 32]
```diff
@@ -112,5 +112,62 @@ Date 13-09-2017
...
+.customLocation {
+  position: relative;
+  /*Don't really need this just for demo styling*/
+  float: none !important;
+  min-width: calc(100%/2 - 8px);
+  width: auto !important;
+}
+
+.customLocation:after {
+    content: " ";
```
Comment (githubwebcontentor) [inline WebApp/Styles/LocationPanel.css]: indent

File: WebApp/Styles/Main.css [hunk 33]
```diff
@@ -255,40 +255,81 @@ header.main .logo{
...
+    /*font-size: 13px;*/
     color: #ffffff;
 }
 .leftNav_panel .leftLinks ul li a:hover{
     background-color: #374c5a;    
     text-decoration: none;
 }
 .leftNav_panel .leftLinks ul li .linkIcon{
-    display: inline-block;
+/*    display: inline-block;
```
Comment (githubwebcontentor) [inline WebApp/Styles/Main.css]: remove comments

File: WebApp/Styles/Main.css [hunk 34]
```diff
@@ -255,40 +255,81 @@ header.main .logo{
...
+
+    display: inline-block;
+    margin: 0 5px 0 3px;
+    width: 27px;
     text-align: center;
+    min-height: 20px;
+    background-image: url(../Images/UI/all-icons.png);
+}
+/* Adding icons from common file */
+.searchICN{
```
Comment (githubwebcontentor) [inline WebApp/Styles/Main.css]: when we use fonts instead of image probably this is not needed.

File: WebApp/Styles/Main.css [hunk 35]
```diff
@@ -664,31 +660,105 @@ label{
...
 }
-    .mapNavigation .selectCol button{
+.mapNavigation .selectCol button{
     background-color: #172936;
     color: #fff;
     height:26px;
 }
 .mapNavigation button {
-    background:#ffffff url("../Images/index.png") no-repeat scroll right center;
+    /*background:#ffffff url("../Images/index.png") no-repeat scroll right center;*/
```
Comment (githubwebcontentor) [inline WebApp/Styles/Main.css]: remove comments

File: WebApp/Styles/Main.css [hunk 36]
```diff
@@ -664,31 +660,105 @@ label{
...
+/*    background-image: url(../Images/UI/all-icons.png);
+    background-position:0 0; 
+    background-repeat: no-repeat;*/
     display:block;
     padding-right:20px;
     border:none; width:100%;
     border-radius: 3px;
     height: 30px;
 }
+/*.dropdown:after{
```
Comment (githubwebcontentor) [inline WebApp/Styles/Main.css]: remove comments

File: WebApp/Styles/Map.css [hunk 37]
```diff
@@ -391,3 +275,76 @@ input.MarkupText {
...
+.leaflet-top {
+  top: 40px;
+  margin-top:6px;
+  float:right;
+}
+.leaflet-right{
+  margin:5% 0.0% 5px 0;
+}
+
+
```
Comment (githubwebcontentor) [inline WebApp/Styles/Map.css:282]: remove extra line

File: WebApp/Styles/MarkupPanel.css [hunk 38]
```diff
@@ -330,7 +332,265 @@ New Style for Markup Panel
...
+  background-color: #ffffff;
+  background-image: none;
+  border: none;
+  -ms-word-break: normal;
+  word-break: normal;
+  outline: none;
+}
+
+
+/*.dataTable tbody tr td:first-child{
```
Comment (githubwebcontentor) [inline WebApp/Styles/MarkupPanel.css]: remove comments

File: WebApp/Styles/Responsive.css [hunk 39]
```diff
@@ -177,20 +188,38 @@ padding: 0;*/
   }
 
   #cmdEmail {
-    width: 25px;
+/*    width: 25px;
```
Comment (githubwebcontentor) [inline WebApp/Styles/Responsive.css]: remove comments

File: WebApp/Styles/Responsive.css [hunk 40]
```diff
@@ -177,20 +188,38 @@ padding: 0;*/
...
     margin: 0 5px;
     padding: 14px 0px;
     text-indent: -9999px;
-    background-position: 19% 50%;
+    background-position: 19% 50%;*/
   }
 
   .helpTxt {
-    width: 25px;
+/*    width: 25px;
```
Comment (githubwebcontentor) [inline WebApp/Styles/Responsive.css]: remove comments

File: WebApp/Styles/Status.css [hunk 41]
```diff
@@ -77,8 +77,8 @@ div
   text-decoration: underline;
 }
 
-#pnlFooter
+/*#pnlFooter
```
Comment (githubwebcontentor) [inline WebApp/Styles/Status.css]: remove comments

File: WebApp/Styles/Status.css [hunk 42]
```diff
@@ -14,10 +14,10 @@ div
   bottom: 0px;
 }
 
-#pnlHeader
+/*#pnlHeader
```
Comment (githubwebcontentor) [inline WebApp/Styles/Status.css]: remove comments

File: WebApp/Styles/Viewer.css [hunk 43]
```diff
@@ -150,27 +141,21 @@ button {
...
 
-#cmdMenu {
+/*#cmdMenu {
   float: left;
   margin: 8px;
-}
+}*/
 
-#pnlFunctionSidebar {
+/*#pnlFunctionSidebar {
```
Comment (githubwebcontentor) [inline WebApp/Styles/Viewer.css]: remove comments

File: WebApp/Styles/Viewer.css [hunk 44]
```diff
@@ -150,27 +141,21 @@ button {
...
-  font-size: 19px;
-  padding: 0 4px;
-}*/
-
 #cmdShowDetails {
   cursor: pointer;
 }
 
-#cmdMenu {
+/*#cmdMenu {
```
Comment (githubwebcontentor) [inline WebApp/Styles/Viewer.css]: remove comments

File: WebApp/Styles/Viewer.css [hunk 45]
```diff
@@ -525,3 +385,128 @@ option .toolsTxt{
...
+    background-position: -46px -11px;
+    display: inline-block;
+    margin-right: 14px;
+}
+#tabMobDetails{
+    display: none!important;
+}
+
+#pnlData{
+    /*background-color: red;*/
```
Comment (githubwebcontentor) [inline WebApp/Styles/Viewer.css]: remove comments

File: WebApp/Viewer.aspx [hunk 46]
```diff
@@ -171,26 +165,29 @@ limitations under the License.
...
                     <span class="txt glyphicon">Identify</span></a></li>
                   <li id="optSelect" runat="server" class="Button MapTool"><a href="#">
-                    <img class="flag " src="Images/select-icon.png" alt="" />
-                    <span class="txt glyphicon">Select</span></a></li>
+                <span class="flag mapToolSelect"></span>
+                    <span class="txt glyphicon">Select</span></a>
+                  </li>
                   <li id="optMarkupTool" runat="server" class="Button MapTool"><a href="#">
-                    <img class="flag " src="Images/draw-icon.png" alt="" />
+                                        <span class="flag mapToolDraw"></span>
```
Comment (githubwebcontentor) [inline WebApp/Viewer.aspx]: indent

File: WebApp/App_Code/BusinessLogic/Configuration.cs (hunk 1 above)
Comment (sub061) [reply WebApp/App_Code/BusinessLogic/Configuration.cs]: done

File: WebApp/BaseMap.ascx.cs (hunk 3 above)
Comment (sub061) [reply WebApp/BaseMap.ascx.cs]: done

File: WebApp/BaseMap.ascx.cs (hunk 4 above)
Comment (sub061) [reply WebApp/BaseMap.ascx.cs]: done

File: WebApp/Development/CompiledIcons.ashx (hunk 5 above)
Comment (sub061) [reply WebApp/Development/CompiledIcons.ashx]: this is for generating all image in single Image file . 
now its change .

File: WebApp/Header.ascx (hunk 6 above)
Comment (sub061) [reply WebApp/Header.ascx]: done

File: WebApp/LegendPanel.ascx (hunk 7 above)
Comment (sub061) [reply WebApp/LegendPanel.ascx]: FunctionHeader used for changing header color and background color.

File: WebApp/LegendPanel.ascx.cs (hunk 8 above)
Comment (sub061) [reply WebApp/LegendPanel.ascx.cs:303]: this code is shifted in baseMap.cs . this is for basemap and overview layer.

Comment (sub061) [reply WebApp/LegendPanel.ascx.cs:303]: now remove this comment

File: WebApp/LocationPanel.ascx (hunk 9 above)
Comment (sub061) [reply WebApp/LocationPanel.ascx]: Added comment for FunctionHeader class

File: WebApp/MarkupPanel.ascx (hunk 10 above)
Comment (sub061) [reply WebApp/MarkupPanel.ascx:17]: Added comment

File: WebApp/MarkupPanel.ascx (hunk 11 above)
Comment (sub061) [reply WebApp/MarkupPanel.ascx]: done

File: WebApp/MarkupPanel.ascx (hunk 12 above)
Comment (sub061) [reply WebApp/MarkupPanel.ascx:125]: this id is used . for updating Markup detail value

File: WebApp/Scripts/BaseLayer.js (hunk 13 above)
Comment (sub061) [reply WebApp/Scripts/BaseLayer.js]: done

File: WebApp/Scripts/BaseLayer.js (hunk 14 above)
Comment (sub061) [reply WebApp/Scripts/BaseLayer.js]: removed

File: WebApp/Scripts/MarkupPanel.js (hunk 15 above)
Comment (sub061) [reply WebApp/Scripts/MarkupPanel.js]: done

File: WebApp/Scripts/MarkupPanel.js (hunk 16 above)
Comment (sub061) [reply WebApp/Scripts/MarkupPanel.js]: done

File: WebApp/Scripts/MarkupPanel.js (hunk 18 above)
Comment (sub061) [reply WebApp/Scripts/MarkupPanel.js]: done

File: WebApp/SelectionPanel.ascx (hunk 19 above)
Comment (sub061) [reply WebApp/SelectionPanel.ascx]: comment added

File: WebApp/SharePanel.ascx (hunk 20 above)
Comment (sub061) [reply WebApp/SharePanel.ascx]: done

File: WebApp/SharePanel.ascx (hunk 21 above)
Comment (sub061) [reply WebApp/SharePanel.ascx:47]: seg_title change to Functionalabel in this file (SharePanel) .  Functionalabel  class used for change color and font size

File: WebApp/Styles/Customize.css (hunk 22 above)
Comment (sub061) [reply WebApp/Styles/Customize.css]: remove

Comment (sub061) [reply WebApp/Styles/Customize.css]: Removed

File: WebApp/Styles/Customize.css (hunk 23 above)
Comment (sub061) [reply WebApp/Styles/Customize.css]: done

File: WebApp/Styles/Customize.css (hunk 24 above)
Comment (sub061) [reply WebApp/Styles/Customize.css]: done

File: WebApp/Styles/DataList.css (hunk 27 above)
Comment (sub061) [reply WebApp/Styles/DataList.css]: done

File: WebApp/Styles/DataList.css (hunk 28 above)
Comment (sub061) [reply WebApp/Styles/DataList.css]: done

File: WebApp/Styles/DataList.css (hunk 29 above)
Comment (sub061) [reply WebApp/Styles/DataList.css]: done

File: WebApp/Styles/LegendPanel.css (hunk 30 above)
Comment (sub061) [reply WebApp/Styles/LegendPanel.css]: done

File: WebApp/Styles/LegendPanel.css (hunk 31 above)
Comment (sub061) [reply WebApp/Styles/LegendPanel.css:56]: done

File: WebApp/Styles/LocationPanel.css (hunk 32 above)
Comment (sub061) [reply WebApp/Styles/LocationPanel.css]: done

File: WebApp/Styles/Main.css (hunk 33 above)
Comment (sub061) [reply WebApp/Styles/Main.css]: done

File: WebApp/Styles/Main.css (hunk 35 above)
Comment (sub061) [reply WebApp/Styles/Main.css]: done

File: WebApp/Styles/Main.css (hunk 36 above)
Comment (sub061) [reply WebApp/Styles/Main.css]: done

File: WebApp/Styles/Map.css (hunk 37 above)
Comment (sub061) [reply WebApp/Styles/Map.css:282]: done

File: WebApp/Development/CompiledIcons.ashx (hunk 5 above)
Comment (githubwebcontentor) [reply WebApp/Development/CompiledIcons.ashx]: Ok Seems Good

File: WebApp/Styles/MarkupPanel.css (hunk 38 above)
Comment (sub061) [reply WebApp/Styles/MarkupPanel.css]: done

File: WebApp/Styles/Responsive.css (hunk 40 above)
Comment (sub061) [reply WebApp/Styles/Responsive.css]: done

File: WebApp/Styles/Status.css (hunk 41 above)
Comment (sub061) [reply WebApp/Styles/Status.css]: done

File: WebApp/Styles/Status.css (hunk 42 above)
Comment (sub061) [reply WebApp/Styles/Status.css]: done

File: WebApp/Styles/Viewer.css (hunk 43 above)
Comment (sub061) [reply WebApp/Styles/Viewer.css]: done

File: WebApp/Styles/Viewer.css (hunk 44 above)
Comment (sub061) [reply WebApp/Styles/Viewer.css]: done

File: WebApp/Styles/Viewer.css (hunk 45 above)
Comment (sub061) [reply WebApp/Styles/Viewer.css]: done

File: WebApp/Viewer.aspx (hunk 46 above)
Comment (sub061) [reply WebApp/Viewer.aspx]: done

**Pull request merged**